GWAS/PCA.ipynb

```diff
@@ -595,19 +695,19 @@
    },
    "outputs": [],
    "source": [
-    "# Calculate Mahalanobis distance per population and report outliers\n",
+    "##### Calculate Mahalanobis distance per population and report outliers\n",
     "[detect_outliers]\n",
     "# Set the probability to remove outliers eg 0.95 or 0.997\n",
     "parameter: prob = 0.997\n",
     "# Robust Mahalanobis to outliers\n",
     "parameter: robust = True\n",
     "parameter: pca_result = path\n",
     "input: pca_result\n",
-    "output: distance=f'{_input:n}.mahalanobis.rds',\n",
-    "        removed_outliers=f'{_input:n}.outliers',\n",
-    "        analysis_summary=f'{_input:n}.analysis_summary.md',\n",
-    "        qqplot_mahalanobis=f'{_input:n}.mahalanobis_qq.png',\n",
-    "        hist_mahalanobis=f'{_input:n}.mahalanobis_hist.png'\n",
+    "output: distance=f'{_input:n}.{name}.mahalanobis.rds',\n",
```

Comment:
okay again let's output to `cwd` folder because this can potentially be used as a separate workflow. The input does not always come from `cwd`